src/lib/structures/Command.ts

```diff
@@ -1,8 +1,7 @@
-import { AliasPiece, AliasPieceOptions, PieceContext } from '@sapphire/pieces';
```

Comment:
Remove lines 378-391

Reply:
Do we want to remove this possibility to customise that, though?

Reply:
It's no longer being read in the constructor, so yes. You removed that code.

Reply:
It is, in line 62.